kubeflow/katib/README.md

```diff
@@ -55,6 +56,33 @@ ks generate katib katib
 ks apply ${KF_ENV} -c katib
 ```
 
+If you want to use Katib not in GKE and you don't have StorageClass for dynamic volume provisioning at your cluster, you have to create persistent volume to bound your persistent volume claim.
```

Comment:
Include a link to k8s documentation: https://kubernetes.io/docs/concepts/storage/persistent-volumes/